testplan/testing/multitest/base.py

```diff
@@ -1310,6 +1313,47 @@ def _run_testsuite_iter(self, testsuite, testcases):
         if teardown_report is not None:
             yield teardown_report, [self.uid(), testsuite.uid()]
 
+    def _get_parent_uids(self, testsuite, testcase):
```

Comment:
Adding a new function I noticed we lack annotations in this module, but a bit puzzled what a testsuite (which is essentially any testsuite decorated class instance) and a testcase (a decorated method of said class) is in terms of a reasonable type.